scripts/cc-cli/src/commands/collectCoins.ts

```diff
@@ -53,7 +55,13 @@ async function collectCoinsAction(options: OptionValues) {
     signer,
     options.burnTxHash,
   );
-  await event.waitForVerification(800_000);
+
+  const blockTime = (api.consts.babe.expectedBlockTime as U64).toNumber();
+  const unverifiedTaskTimeout = Number(
+    api.consts.creditcoin.unverifiedTaskTimeout.toString(),
```

Comment:
doesn't `as U64).toNumber()` work here? Same as previous line.

Reply:
No, but you can cast it as a U32. I'll push that change now